### PR TITLE
Enable OTLP exporter and add OTLP E2E tests

### PR DESCRIPTION
**Pull request by tigrannajaryan**
- OTLP exporter is now a component enabled in the default build.
- Added OTLP E2E tests for traces.

TODO: Add OTLP E2E tests for metrics once metrics are supported
in OTLP exporter.

Resolves: https://github.com/open-telemetry/opentelemetry-collector/issues/668

### CODE REVIEW

File: testbed/go.sum [hunk 1]
```diff
@@ -265,8 +266,8 @@ github.com/golangci/revgrep v0.0.0-20180526074752-d9c87f5ffaf0 h1:HVfrLniijszjS1
 github.com/golangci/revgrep v0.0.0-20180526074752-d9c87f5ffaf0/go.mod h1:qOQCunEYvmd/TLamH+7LlVccLvUH5kZNhbCgTHoBbp4=
 github.com/golangci/unconvert v0.0.0-20180507085042-28b1c447d1f4 h1:zwtduBRr5SSWhqsYNgcuWO2kFlpdOZbP0+yRjmvPGys=
 github.com/golangci/unconvert v0.0.0-20180507085042-28b1c447d1f4/go.mod h1:Izgrg8RkN3rCIMLGE9CyYmU9pY2Jer6DgANEnZ/L/cQ=
-github.com/google/addlicense v0.0.0-20190510175307-22550fa7c1b0 h1:ydbHzabf84uucKri5fcfiqYxGg+rYgP/zQfLLN8lyP0=
-github.com/google/addlicense v0.0.0-20190510175307-22550fa7c1b0/go.mod h1:QtPG26W17m+OIQgE6gQ24gC1M6pUaMBAbFrTIDtwG/E=
+github.com/google/addlicense v0.0.0-20200301095109-7c013a14f2e2 h1:N/H6+jLLYNoskSbyRaph9ttbcaGEw4AWpAZw+1VqIGs=
+github.com/google/addlicense v0.0.0-20200301095109-7c013a14f2e2/go.mod h1:EMjYTRimagHs1FwlIqKyX3wAM0u3rA+McvlIIWmSamA=
```
Comment (bogdandrutu) [inline testbed/go.sum]: This looks unrelated to the change, maybe a rebase helps?

Comment (tigrannajaryan) [reply testbed/go.sum]: This is after running `go mod tidy`. Looks like it was not tidied up previously and now this change is needed.

Comment (bogdandrutu) [reply testbed/go.sum]: ahhh... this is in testbed :(...

**Comment by bogdandrutu** [comment]
Please use branches in your fork instead of on the master :)

**Comment by tigrannajaryan** [comment]
> Please use branches in your fork instead of on the remote repo :)

Right. I accidentally pushed to the wrong remote.
